scripts/aws/cfn/microservice.yml

```diff
@@ -155,7 +152,7 @@ Resources:
         S3ObjectVersion: !Ref LambdaZipVersionId2
       Environment:
         Variables:
-          KEY_ID: !Ref JwtEnvironmentKeyId
+          KEY_ALIAS: !Ref SessionTokenSigningKeyAlias
```

Comment:
Rinominiamo la chiave in PnAuthFleetJwtSignKeyAlias (se non ricordo male c'è anche un prefisso che permette il deploy di due ambienti sullo stesso account/region)